src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.props

```
@@ -20,6 +20,8 @@ Copyright (c) .NET Foundation. All rights reserved.
   <PropertyGroup>
     <!-- Disable web SDK implicit package versions for ASP.NET packages, since the .NET SDK now handles that -->
     <EnableWebSdkImplicitPackageVersions>false</EnableWebSdkImplicitPackageVersions>
+    <!-- Unconditionally use runtime packs for NativeAOT on .NET 9.0 or higher -->
+    <PublishAotUsingRuntimePack Condition="'$(_IsPublishing)' == 'true' And '$(PublishAot)' == 'true' And ('$(_TargetFrameworkVersionWithoutV)' != '') And ('$(_TargetFrameworkVersionWithoutV)' >= '9.0')">true</PublishAotUsingRuntimePack>
```

nit, nearby files are using MSBuild prop function for version comparisons:
```suggestion
    <PublishAotUsingRuntimePack Condition="'$(_IsPublishing)' == 'true' And '$(PublishAot)' == 'true' And $([MSBuild]::VersionGreaterThanOrEquals($(_TargetFrameworkVersionWithoutV), '9.0'))">true</PublishAotUsingRuntimePack>
```

---

I copied the condition from the code below in the same file...

---

Can you please add a check for PublishAotUsingRuntimePack = '' so there's a way to opt-out until we're confident it works?

---

Yep